ngrinder-controller/src/main/java/org/ngrinder/script/model/GitHubConfig.java

```diff
@@ -65,5 +67,34 @@ public GitHubConfig deserialize(JsonParser jsonParser, DeserializationContext ct
 		private String defaultIfNull(JsonNode jsonNode, String defaultValue) {
 			return jsonNode == null ? defaultValue : jsonNode.asText(defaultValue);
 		}
+
+		/**
+		 * Check base-url configuration value conforms to url format.
+		 *
+		 * @throws InvalidGitHubConfigurationException occurs when Not conform to url format.
+		 * */
+		private void checkUrlFormat(String baseUrl) {
+			UrlValidator urlValidator = getInstance();
+			if (!baseUrl.isEmpty() && !urlValidator.isValid(baseUrl)) {
+				throw new InvalidGitHubConfigurationException("Field 'base-url' doesn't conform to url format.\nPlease check your .gitconfig.yml");
+			}
+		}
+
+		/**
+		 * Check gitHub configuration has unsupported fields.
+		 *
+		 * @throws InvalidGitHubConfigurationException occurs when unsupported field is exist.
+		 * */
+		private void checkUnsupportedField(JsonNode jsonNode) {
+			Iterator<Entry<String, JsonNode>> fields = jsonNode.fields();
+			while (fields.hasNext()) {
+				Entry<String, JsonNode> field = fields.next();
+				String key = field.getKey();
+				if (!isSupportedField(key)) {
```

Comment:
Notify error message  if user set unsupported field in github configuration file

![image](https://user-images.githubusercontent.com/14273601/82900490-95c7d180-9f97-11ea-86d2-9abb5ea7a45f.png)

Reply:
good.